### DISCUSSION TITLE
Filter out problematic geometries

### DISCUSSION THREAD

**Discussion by outpark** [transitland/transitland#398] [Q&A] [answered]
<img width="833" alt="Screenshot 2023-06-28 at 10 08 39 PM" src="https://github.com/transitland/transitland/assets/12371511/e58ab475-b55b-4013-8933-cf1fcb9b1df8">
The global map example has an option to filter out the problematic geometries - https://www.transit.land/news/2022/06/10/vector-map-tiles#problematic-route-geometries.

But they are included when I use the vector tiles (https://transit.land/api/v2/tiles/routes/tiles/). Is there a way to easily turn it off like the example map? I couldn't find this anywhere and the transit data is virtually useless without this option (viewing New York area)

**Comment by drewda** [comment, marked as answer]
Here's the logic used by the global map to filter out problematic geometries: https://github.com/interline-io/tlv2-ui/blob/main/src/components/map-viewer.vue#L171-L175

**Reply by outpark** [reply]
Thank you!!
